pontoon/base/tests/test_views.py

```diff
@@ -123,6 +123,34 @@ def test_not_authed_public_project(self):
         # I'd assertTemplateUsed here but it doesn't work on non-DTL
         # templates.
 
+    def test_locale_agnostic_user_authed(self):
+        """
+        User is authenticated and Userprofile.custom_homepage defined,
+        redirect to Userprofile.custom_homepage
+        """
+        ProjectFactory.create(slug='valid-project')
+        LocaleFactory.create(code='custom-locale')
```

Comment:
A locale must be enabled for a project and project must have at least one resource for translate view to work. So you'd need to change the set up into something like this:

```python
locale = LocaleFactory.create(code='custom-locale')
project = ProjectFactory.create(slug='valid-project', locales=[locale])
ResourceFactory.create(project=project)
```